### ISSUE TITLE
Proposal to Limit DID Method Selection to One Each From These Categories: Ephemeral, Web, Fully Decentralized

### ISSUE THREAD

**Issue by jrayback** [decentralized-identity/did-methods#25]
This suggestion is mentioned in our charter and has been informally discussed since the formation of the group. Can we formally decide that the output of this working group will be the selection & standardization of exactly three DID Methods - one ephemeral, one web-based, and one that is fully decentralized?

**Comment by ottomorac** [comment]
Hello I beg to differ. I know of at least 3 great did methods that are decentralized and it would not be fair to limit participation to just one (did:cheqd, did:iden3, and did:ebsi). As mentioned in our call earlier this week is to create a more general ranking of did methods overall.

**Comment by mwherman2000** [comment]
Part of my proposal is to group related proposed DID Methods into _clusters_ ...then pick a representative from each cluster.
This sound IMO to be more inclusive (i.e. don't create "haves" and "have nots").  Every DID Method has a home (i.e. it's cluster) and one of the methods will becomes a guide/North Star for the other members of the cluster. 

[I'll add a list of the detailed issues that discuss clustering further]

**Comment by jrayback** [comment]
I support the suggestion that we endorse a peer DID method as well. I understand that stating these three categories doesn't limit us _per se_ but I think that stating explicitly that we want a peer DID makes sure that we select one.

**Comment by ottomorac** [comment]
As stated in the meeting on 15-Jan. A different approach might be to cap the number of did methods at 10 total. That type of suggestion also sounds reasonable to me. I agree with the principle that for this initial version of the spec we do not want a super long list of did methods that would slow us down as a working group.

**Comment by peacekeeper** [comment]
> we do not want a super long list of did methods that would slow us down

Yes.. Let's see how many proposal we get, and how many contributors.

**Comment by msporny** [comment]
> A different approach might be to cap the number of did methods at 10 total. 

I'll note that trying to standardize 10 DID methods, of even 5 via a global standards setting organization (like the W3C or ISO) would be a really heavy lift... it would take years, even if we had the specs more or less complete by the time we started the global standardization work. 

Generally speaking, if we have more than 3, we need a strategy to use multiple SDOs and multiple SDO teams to get the specs through the long processes. I'm not saying it can't be done... just noting how big of a task getting a global standard done properly is.
